frontend/src/components/NavBar/styles.js

```diff
@@ -33,6 +33,33 @@ const styles = theme => ({
   h5: {
     marginBottom: theme.spacing(2),
   },
+  container: {
```

Comment:
This shouldn't have the name `container` as it seems too general. Maybe a better name would be `socialContainer`.

Reply:
Added!